specification/components/schemas/extensions/Extension-eRS-ReferralState.yaml

```diff
@@ -77,7 +77,7 @@ properties:
                   type: string
                   enum:
                     - __baseurl__/FHIR/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON
-                    - __baseurl__/FHIR/STU3/CodeSystem/REQUEST-CANCELLATION-REASON
+                    - __baseurl__/FHIR/STU3/CodeSystem/REFERRAL-CANCELLATION-REASON
```

Comment:
@nhsd-jack-wainwright I can see double underscore prefix and suffix to baseurl. Is that new change coming? Also noticed the /FHIR/ path in the code system URL which is not seen other code system url.

Reply:
Nope good spot 🙂 , this should be `_basturl_/STU3/CodeSystem/REFERRAL-CANCELLATION-REASON`.